### PR TITLE
Add Python strings strip function documentation

### PR DESCRIPTION
**Pull request by laisvigas**
Description

Closes #<3365>

I have added documentation for the Python method strip() to the project's documentation. I explained how the method works using examples. My motivation was to contribute to the Codecademy repository, as Codecademy has played a significant role in developing my programming skills. Through this work, I was able to enhance my understanding of Git and GitHub.

Type of Change

* Adding a new entry
* Updating the documentation

Checklist
* [*] All writings are my own.
* [*] My entry follows the Codecademy Docs style guide.
* [*] My changes generate no new warnings.
* [*] I have performed a self-review of my own writing and code.
* [*] I have checked my entry and corrected any misspellings.
* [*] I have made corresponding changes to the documentation if needed.
* [*] I have confirmed my changes are not being pushed from my forked main branch.
* [*] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
* [*] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes # in the "Description" above).

### CODE REVIEW

**Comment by laisvigas** [comment]
Hi, @KTom101 

I followed the instructions to create a new entry for the 'strip' function, but I am encountering these warnings, and I don't understand if there's something wrong. I even started over, but I'm not sure exactly what I should do from now on. Could you please guide me on this? Thank you!

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/strip/strip.md:3]:
```suggestion
Description: 'Eliminates spaces at the beginning and end of a string, and specified characters when passed as an argument.'
```

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/strip/strip.md:24]: Generally we explain the parameters here. Briefly explain what `characters` takes/does

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/strip/strip.md:67]:
```suggestion
The following code is runnable and uses the `.strip()` method.
```

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/strip/strip.md:45]:
```suggestion
```

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/strip/strip.md:63]:
```suggestion
```

File: content/python/concepts/strings/terms/strip/strip.md
Comment (caupolicandiaz) [reply content/python/concepts/strings/terms/strip/strip.md:45]: For most simple entry examples we don't include additional text explaining the output. Additional explanations are generally reserved for concept entries that include more material, or code blocks that may contain complex or unintuitive code.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@laisvigas I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by laisvigas** [comment]
Hi, @caupolicandiaz

Thank you for your feedback. I have completed the review and made the necessary changes. Please feel free to let me know if there's anything else I can do or if there are further improvements you'd like to suggest. I appreciate your time and input.

**Review by caupolicandiaz — APPROVED** [review]
@laisvigas I just made a few direct edits to correct some of the spacing. This is ready for a second review.

**Comment by laisvigas** [comment]
Thank you for your prompt review and for making the necessary spacing corrections, @caupolicandiaz . Since I am new to this, I would like to ask if I should do anything regarding the minor changes, considering my working tree is up to date, or should I just wait for the second review?

**Comment by KTom101** [comment]
@laisvigas You only have to wait for a second review.

File: content/python/concepts/strings/terms/strip/strip.md
Comment (KTom101) [inline content/python/concepts/strings/terms/strip/strip.md:3]:
```suggestion
Description: 'Eliminates any trailing spaces at the beginning and end of a string. Specific characters can be passed in as an argument to be removed instead.'
```

File: content/python/concepts/strings/terms/strip/strip.md
Comment (KTom101) [inline content/python/concepts/strings/terms/strip/strip.md:16]: A couple of changes here. Let's also include a note further explaining when specified characters are removed.
```suggestion
The **`.strip()`** method removes any whitespace at the beginning and end of a string. Optionally, a string of characters can be passed in as an argument to remove those characters instead. Otherwise, all whitespaces will be removed.

> **Note**: The .strip() method removes specified characters only when they appear as the outermost leading and trailing characters in the string.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@laisvigas I've added some changes.

**Comment by laisvigas** [comment]
Hi, @KTom101

I just reviewed the changes. Please let me know if everything is all right. Thanks!

**Review by KTom101 — APPROVED** [review]

**Pull request merged**
